lib/rules/no-callback-leaks-in-ember-objects.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+// const ember = require('../utils/ember');
+const utils = require('../utils/utils');
+
+
+//------------------------------------------------------------------------------
+// Ember object rule - Avoid callback memory leaks
+// Callback leaks are memory leaks that occur due to state being caught
+// in a callback function that is never released from memory.
+//------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Avoids callback memory leaks',
+      category: 'Ember Object',
+      recommended: true,
+      url: 'https://github.com/ember-best-practices/memory-leak-examples/blob/master/exercises/exercise-2.md'
```

Comment:
I would just remove this `url` property. It should either be left out or it should link to the actual rule documentation.